src/hooks/TestUseLogin.tsx

```diff
@@ -0,0 +1,62 @@
+// src/hooks/TestUseLogin.tsx
+'use client';
+
+import React, { useEffect, useState } from 'react';
+import useLogin from '@/hooks/useLogin';
+
+const TestUseLogin: React.FC = () => {
+  const { user, error, loading, login } = useLogin();
+  const [email, setEmail] = useState('email08@te978sting.com'); // Default valid email
+  const [password, setPassword] = useState('teste4test'); // Default valid password
+
+  const handleLogin = () => {
+    login(email, password);
+  };
+
+  useEffect(() => {}, [login]);
+
+  return (
+    <div>
+      <h2>Test Login Hook</h2>
+      <div>
+        <label>Email:</label>
+        <input
+          type="text"
```

Comment:
no change needed here and let moe this but for type there is a type="email" in HTML which comes with email validation ready out of the box

Reply:
Nice!!! thats def will help Moe out a  bunch, I let him know